src/index.js

```diff
@@ -2,0 +1,43 @@
+// 'use strict';
+
+// import express from 'express';
+// import cors from 'cors';
+
+// const app = express();
+
+// app.use(cors({ origin: '*' }));
+// app.use(express.json());
+
+// const messages = [];
+
+// app.get('/', (req, res) => {
```

Comment:
The route handler for the root path is commented out. Uncomment these lines to respond with 'Hello World' when accessing the root URL.